packages/agreement-process/src/model/domain/errors.ts

```diff
@@ -277,3 +281,24 @@ export function attributeNotFound(
     title: "Attribute not found",
   });
 }
+
+export function delegationNotFound(
+  delegationId: DelegationId
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `Delegation ${delegationId} not found`,
+    code: "delegationNotFound",
+    title: "Delegation not found",
+  });
+}
+
+export function missingDelegationId(
+  organizationId: TenantId,
+  eserviceId: EServiceId
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `Missing delegation id for organization ${organizationId} and EService ${eserviceId}`,
```

Comment:
A suggestion to make the error clearer
```suggestion
    detail: `An Active delegation exists for EService ${eserviceId} where requester organization ${organizationId} is delegator or delegate, but delegation id is missing`,
```